app/src/validators/permission.js

```diff
@@ -0,0 +1,60 @@
+const { validate, Joi } = require('express-validation');
+const { uuidv4, uuidv4MultiModel, permCodeMultiModel } = require('./common');
+const { Permissions } = require('../components/constants');
+
+const schema = {
+  searchPermissions: {
+    query: Joi.object({
+      userId: uuidv4MultiModel,
+      objId: uuidv4MultiModel,
+      permCode: permCodeMultiModel
+    }).min(1)
```

Comment:
Was this min(1) intentional?
providing no query parameters does return a result from the API (all the records in the object_permissions table)
then again maybe it's nice to make the search more prescriptive, i dont see anyone needing the whole list for all users and objects..